services/tenant-ui/frontend/src/helpers/constants.ts

```diff
@@ -62,13 +62,13 @@ export const API_PATH = {
   INNKEEPER_RESERVATIONS_DENY: (id: string) =>
     `/innkeeper/reservations/${id}/deny`,
 
-  ISSUE_CREDENTIAL_RECORDS: 'issue-credential/records',
-  ISSUE_CREDENTIAL_RECORD: (id: string) => `issue-credential/records/${id}`,
+  ISSUE_CREDENTIAL_RECORDS: '/issue-credential/records',
+  ISSUE_CREDENTIAL_RECORD: (id: string) => `/issue-credential/records/${id}`,
   ISSUE_CREDENTIAL_RECORDS_SEND_OFFER: (id: string) =>
-    `issue-credential/records/${id}/send-offer`,
-  ISSUE_CREDENTIALS_SEND_OFFER: 'issue-credential/send-offer',
+    `/issue-credential/records/${id}/send-offer`,
```

Comment:
Will changing these impact the API calls? OR have they been tested as well?

Reply:
I did test this by putting an invalid path and then with/without the slash. I did that a long time ago though so I can test it again.